debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/SourceInfo.java

```diff
@@ -28,49 +27,26 @@
     public static final String SNAPSHOT_KEY = "snapshot";
     public static final String TIMESTAMP_KEY = "ts_micro";
 
-    public static final Schema SOURCE_SCHEMA = SchemaBuilder.struct().name(Record.SOURCE)
-            .field(DEBEZIUM_VERSION_KEY, Schema.STRING_SCHEMA)
-            .field(DEBEZIUM_CONNECTOR_KEY, Schema.STRING_SCHEMA)
-            .field(CLUSTER_KEY, Schema.STRING_SCHEMA)
-            .field(COMMITLOG_FILENAME_KEY, Schema.STRING_SCHEMA)
-            .field(COMMITLOG_POSITION_KEY, Schema.INT32_SCHEMA)
-            .field(SNAPSHOT_KEY, Schema.BOOLEAN_SCHEMA)
-            .field(KEYSPACE_NAME_KEY, Schema.STRING_SCHEMA)
-            .field(TABLE_NAME_KEY, Schema.STRING_SCHEMA)
-            .field(TIMESTAMP_KEY, CassandraTypeKafkaSchemaBuilders.TIMESTAMP_MICRO_TYPE)
-            .build();
-
     public final String version = Module.version();
     public final String connector = Module.name();
-    public final String cluster;
-    public final OffsetPosition offsetPosition;
-    public final KeyspaceTable keyspaceTable;
-    public final boolean snapshot;
-    public final long tsMicro;
+    public String cluster;
+    public OffsetPosition offsetPosition;
+    public KeyspaceTable keyspaceTable;
+    public boolean snapshot;
+    public long tsMicro;
```

Comment:
This should be `Instant` better. We try to convert any timestamps to `Instant` right at the system boundary as early as possible and then use that more meaningful type internally. Only where needed, it's "rendered" as long timestamp again.

Reply:
@bingqinzhou could you still address this remark?

Reply:
@gunnarmorling Updated accordingly.